keps/sig-api-machinery/2876-crd-validation-expression-language/README.md

```diff
@@ -589,6 +590,28 @@ associativeList.all(e, e.key1 == 'a' && e.key2 == 'b' && e.val == 100)
 associativeList.all(e, e.val == 100)
 ```
 
+### Resource constraints
+
+For Beta, per-request execution time will be constrained via a context-passed timeout. For Beta, we are looking at a range between 100 milliseconds and 1 second for the timeout. We want to see how
+CEL performs in Beta before deciding what specific duration to set the timeout to. If the timeout is 
```

Comment:
nit: Instead of "performs in Beta", say "performs in benchmarks once our Beta optimizations are in place" ?

Reply:
Makes more sense that way for sure, fixed